assets/dev/js/frontend/handlers/accessibility/nested-title-keyboard-handler.js

```diff
@@ -1,6 +1,4 @@
-import Base from '../base';
```

Comment:
The module approach, requires us to load all the imports as script modules as well.